src/tests/ftest/dfuse/find.py

```diff
@@ -193,54 +192,58 @@ def _search_needles(file_name, sample_tag, expected_res):
 
         return profiler
 
-    def _setup_containers(self, dfs_path, cont_count, dfuses, containers):
+    def _setup_containers(self, pool, dfs_path, cont_count, dfuses, containers):
         """
         Setup as many containers as the test requested. Return the paths where
         the containers were mounted.
         """
         mount_dirs = []
         for count in range(cont_count):
-            self.add_container(self.pool)
-            cont_dir = "{}_daos_dfuse_{}".format(self.pool.uuid, count)
+            container = self.get_container(pool)
+            cont_dir = "{}_daos_dfuse_{}".format(pool.uuid, count)
             mount_dir = os.path.join(dfs_path, cont_dir)
-            self.log.info(
-                "Creating container Pool UUID: %s Con UUID: %s", self.pool, self.container)
+            self.log.info("Creating container Pool UUID: %s Con UUID: %s", pool, container)
             dfuse = get_dfuse(self, self.hostlist_clients)
-            start_dfuse(self, dfuse, pool=self.pool, container=self.container, mount_dir=mount_dir)
+            start_dfuse(self, dfuse, pool, container, mount_dir=mount_dir)
             dfuses.append(dfuse)
-            containers.append(self.container)
+            containers.append(container)
             mount_dirs.append(dfuse.mount_dir.value)
 
         return mount_dirs
 
     def _create_dir_forest(self, paths, height, subdirs, files_per_node, needles):
         """Create a directory tree on each path listed in the paths variable"""
-        remote_pythonpath = ":".join(sys.path)
-
         count = 0
         for path in paths:
-            self.log.info("Populating: %s", path)
-            prefix = "t{:05d}_".format(count)
+            dir_tree = DirectoryTreeCommand(self.hostlist_clients)
+            dir_tree.path.value = path
+            dir_tree.height.value = height
+            dir_tree.subdirs.value = subdirs
+            dir_tree.files.value = files_per_node
+            dir_tree.needles.value = needles
+            dir_tree.prefix.value = "t{:05d}_".format(count)
```

Comment:
FYI I think this is the f-string way
```suggestion
            dir_tree.prefix.value = f"t{count:05d}_".
```